xwiki-platform-core/xwiki-platform-legacy/xwiki-platform-legacy-office-importer/src/main/java/org/xwiki/officeimporter/internal/openoffice/DefaultOpenOfficeConverter.java

```diff
@@ -43,17 +42,6 @@
      */
     private final OfficeConverter converter;
 
-    /**
-     * Creates a new {@link DefaultOfficeConverter} instance.
-     * 
-     * @param converter provided by JODConverter library
-     * @param workDir space for holding temporary file
-     */
-    public DefaultOpenOfficeConverter(OfficeDocumentConverter converter, File workDir)
```

Comment:
This one is a legacy constructor I had to remove because the `OfficeDocumentConverter` type is no longer available in the new jodconverter. How do we deal with such case?

Reply:
Not sure what you mean ? It's an internal class so retro-compatibility depends on the mood.

Reply:
> It's an internal class so retro-compatibility depends on the mood.

ok I wanted to ensure it was ok :)